### DISCUSSION TITLE
Is there a way to add a torrent referring to an existing file and automatically start it only if verification succeeds?

### DISCUSSION THREAD

**Discussion by GordonFreemanK** [transmission/transmission#8490] [Q&A]
### What is the issue?

So let's say I do an RPC call for `add-torrent`, and the file(s) is/are found on the drive by Transmission, but Transmission decides that it needs to verify the data, either because `torrent-added-verify-mode` is set to `full` or because the `fast` mode [decides verification is needed](https://github.com/transmission/transmission/pull/2626).

Is there a way to configure Transmission so that if verification fails, the torrent is automatically paused (if `started` was set to `true` on the RPC call)?

I am currently working with a tool that automatically seeds existing files, but some of these files might have incorrect data - so I want to start the torrents if possible, but also make sure that existing data isn't overwritten if verification fails.

### Which application of Transmission?

None

### Which version of Transmission?

4.x.x (e.g. 4.0.5)

**Comment by killemov** [comment]
There is some manual action required in any solution. Main thing is you should always add those torrents paused no matter what. Then,  at a later date, you can decide to start a torrent depending on `percentDone`. You might be able to script that last part.

**Comment by GordonFreemanK** [comment]
Thanks for your reply! It would be nice to have a way to control this.

The code already has [some awareness](https://github.com/transmission/transmission/blob/9f77ef9c7ae32c1dcb7be79fca338f3ea1fb62fe/libtransmission/torrent.cc#L831) of the distinction between a torrent added for downloading and a torrent added for seeding. It would be good to prevent torrents added for seeding (i.e. basically if there's pre-existing data?) from being started when the verification is triggered and ultimately fails.
